test/runtests.jl

```diff
@@ -19,8 +19,11 @@
 
 ### Code:
 
+module PolynomialRootsTests
```

Comment:
I don't think this module is necessary, is it?

Reply:
You are absolutely right, this is just a matter of taste. Sometimes I find it more convenient when batch testing via include("runtests.jl") since it doesnt leave any bindings in the `Main` scope. Should I remove it?

Reply:
I see.  Ok, I think it's fine then to keep it.  Thanks for your work!